Mathlib/Order/Atoms.lean

```diff
@@ -315,9 +325,9 @@ namespace IsCoatomic
 
 variable [OrderTop α] [IsCoatomic α]
 
-instance isCoatomic : IsAtomic αᵒᵈ :=
+instance isAtomic_dual : IsAtomic αᵒᵈ :=
```

Comment:
Surely it should be `OrderDual.instIsAtomic`?

Reply:
I'm following [IsAtomic.isCoatomic_dual](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Atoms.html#IsAtomic.isCoatomic_dual) and [IsCoatomistic.isAtomistic_dual](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Atoms.html#IsCoatomistic.isAtomistic_dual) etc.
If you want to rename all these, maybe open a separate PR, and I'm not sure if a Zulip discussion/vote would be necessary for a maintainer to approve it.

Reply:
I think all those names are pretty clearly against the convention, but sure I'll open a separate PR. Please remove this change from yours to avoid conflicts

Reply:
Now in #13643